### ISSUE TITLE
Upgrade to 32.0.0 causes parseError with media query in .less file

### ISSUE THREAD

**Issue by PinkChampagne17** [stylelint/stylelint-config-standard#287]
### What steps are needed to reproduce the bug?

1. Upgrade the `stylelint-config-standard` package from version `31.0.0` to `32.0.0`.
2. Type the following code block in `test.less` file:
```less
div {
  @print : ~'print';
  @media @print {
    color: black;
  }
}
```
3. Run `stylelint test.less`.

### What did you expect to happen?

Before upgrading the stylelint-config-standard package to 32.0.0, stylelint will not throw an error when using the given code block in a .less file.

### What actually happened?

```
test.less
 4:10  ✖  Cannot parse media query  parseError

1 problem (1 error, 0 warnings)
```

**Comment by ybiquitous** [comment]
@PinkChampagne17 Thanks for the report. I'm trying to reproduce the error on [our demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIoEsBuACYAdAdu9AAgA4BOyOALuvOgH4Dkp5F9A3LvgQLYwoCGhJpUwd86SABsIJGgCMJfMAGt2edAF9c6kABoQAM2QSYAOT48EIGAA9zRYwDpjAZ2e7wEHIYDml7GqwrawoYHChnQJoAbVF8QOcKAE9jCWYAWkgvZG80hL4wvhIoQNEAXR1RQLAAVwSILgBlRMo+a0j0QKIIBLBXNJcIkC13TJ8AMWkuPgpLACtnT3dYIjdEfziQBOSYVMp2wIUQhMCKgM2klPTR7NyKfKhC4oQOkEOYY91KkC6evoH917Td4UEo4bTqIA), but it doesn't occur. Can you provide additional info for reproduction?

**Comment by PinkChampagne17** [comment]
> @PinkChampagne17 Thanks for the report. I'm trying to reproduce the error on [our demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIoEsBuACYAdAdu9AAgA4BOyOALuvOgH4Dkp5F9A3LvgQLYwoCGhJpUwd86SABsIJGgCMJfMAGt2edAF9c6kABoQAM2QSYAOT48EIGAA9zRYwDpjAZ2e7wEHIYDml7GqwrawoYHChnQJoAbVF8QOcKAE9jCWYAWkgvZG80hL4wvhIoQNEAXR1RQLAAVwSILgBlRMo+a0j0QKIIBLBXNJcIkC13TJ8AMWkuPgpLACtnT3dYIjdEfziQBOSYVMp2wIUQhMCKgM2klPTR7NyKfKhC4oQOkEOYY91KkC6evoH917Td4UEo4bTqIA), but it doesn't occur. Can you provide additional info for reproduction?

Sorry I cannot reproduce it using [stylelint demo](https://stylelint.io/demo/), you can get the minimal reproduction in [this repo](https://github.com/p-dustbin/stylelint-config-standard-issue-287).

**Comment by ybiquitous** [comment]
Thank you for providing the reproduction repo! I found this error occurs due to the `media-feature-range-notation` rule.

Fortunately, this error has been fixed in the HEAD revision, so the next version of Stylelint will not raise it (we'll release it soon). See https://github.com/stylelint/stylelint/pull/6760
